packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -293,6 +293,11 @@ export abstract class BucketBase extends cdk.Construct implements IBucket {
    */
   protected abstract autoCreatePolicy = false;
 
+  /**
+   * Whether all block public access settings are enabled
+   */
+  protected abstract publicAccessBlocked?: boolean;
```

Comment:
Since this is not really accurate for the case of an imported bucket, rename this to `disallowPublicAccess`. It will semantically reflect what this is actually used for instead falsely representing this configuration for imported buckets.

Reply:
Is indeed better, done.